### PR TITLE
changed instances of awk -f with cut -d

### PR DESCRIPTION
**Pull request by Aeres-u99**
The only instance of awk remaining is the NR==1 instance, which I will be focussing on next. I am yet to test it and the PR is created for reference only, will request once done with testing for merge

### CODE REVIEW

**Review by hwittenborn — CHANGES_REQUESTED** [review]
Could you remove the changes from the `.drone` folder? Those scripts are using for my CI system, and are irrelevant to the commands used in the built packages.

**Review by hwittenborn — CHANGES_REQUESTED** [review]
Also, any spots that contain a `: ` (contains a space) should not be converted to `:` (which likewise doesn't).

I found the issue in the following files, but I'd double check to make sure I didn't miss anything:

- `src/functions/fakeroot_build.sh`
- `src/functions/get_variables.sh`

File: src/functions/fakeroot_build.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ fakeroot_build() {
         done
 
         field() {
-            cat "DEBIAN/control" | grep "${1}:" | awk -F": " '{print $2}'
+            cat "DEBIAN/control" | grep "${1}:" | cut -d":" -f 2
```
Comment (hwittenborn) [inline src/functions/fakeroot_build.sh]: As well as with the previous note (changing `:` to `: `), could you also add a space between `-d` and `"` like you did with the other files?

File: src/functions/remove_dependency_description.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ remove_dependency_description() {
     # it didn't have a purpose.
     NUM=0
     while [[ "${optdepends[$NUM]}" != "" ]]; do
-        local temp_new_optdepends+=" $(echo ${optdepends[$NUM]} | awk -F: '{print $1}')"
+        local temp_new_optdepends+=" $(echo ${optdepends[$NUM]} | cut -d':' -f 1)"
```
Comment (hwittenborn) [inline src/functions/remove_dependency_description.sh]: Likewise do the same here.

**Review by hwittenborn — CHANGES_REQUESTED** [review]

File: src/functions/fakeroot_build.sh (hunk 1 above)
Comment (Aeres-u99) [reply src/functions/fakeroot_build.sh]: sure sure will look into it and get this sorted.
Thanks a lot for testing and apologies for delays from my end, have been caught up at work lately.

**Comment by Aeres-u99** [comment]
Should be fixed now, testing on my side for nnn and starship went well.

**Comment by hwittenborn** [comment]
Could you pipe the output of the commands you changed to `cut` into `tr -d` so extra whitespace characters get removed?

Normally this could be done with `awk`, but cut only accepts a single character as a delimiter.

**Comment by hwittenborn** [comment]
On second though, could you use `sed 's|^ ||'` instead? Would fix issues with package descriptions for packages in `optdepends` from getting messed up.

I plan on deprecating package descriptions (again, for packages in `optdepends`) anyway, but I don't want to make any breaking changes until that point.

**Comment by Aeres-u99** [comment]
Done.

**Review by hwittenborn — APPROVED** [review]

**Comment by hwittenborn** [comment]
I'm gonna go ahead and close this for now.

If we can ever find a viable way to implement everything, we can just open up a new PR.

**Pull request closed without merging**